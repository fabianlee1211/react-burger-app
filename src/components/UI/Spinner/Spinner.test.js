import React from 'react';
import { shallow } from 'enzyme';
import Spinner from './Spinner';
import { type } from 'os';
import { finished } from 'stream';

describe('<Spinner />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Spinner />);
  });

  it('renders Spinner correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('renders with class Loader', () => {
    expect(wrapper.find('div.Loader')).toHaveLength(1);
  });

  it('renders Loading text', () => {
    expect(wrapper.text()).toBe('Loading...');
  });
});
