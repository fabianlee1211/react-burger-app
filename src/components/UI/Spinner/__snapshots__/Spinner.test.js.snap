// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> renders Spinner correctly 1`] = `
<div
  className="Loader"
>
  Loading...
</div>
`;
